chronicle/release/releasers/github/gh_pull_request.go

```diff
@@ -317,14 +322,28 @@ func fetchMergedPRs(user, repo string) ([]ghPullRequest, error) {
 		}
 
 		// var limit rateLimit
```

Comment:
Is this meant to be commented out? (I see it's not part of this diff, so not a blocking comment; just surprised.)

Reply:
yeah, this is a partial implementation of https://github.com/anchore/chronicle/issues/9